src/components/ArticlesList.jsx

```diff
@@ -0,0 +1,35 @@
+import ArticleListCard from "./ArticleListCard";
+import { getArticles } from "./utils/utils";
+import { useState, useEffect } from "react";
+
+const ArticleList = () => {
+  const [articles, setArticles] = useState("");
+  const [isLoading, setIsLoading] = useState(true);
+  const [isError, setIsError] = useState(false);
+  useEffect(() => {
```

Comment:
The default value of 'isLoading' could be 'false' , then could be set to true here, right before sending a request to the API. This will make sure that the 'loading' message will show up later as well, whenever you fetch them from the API again.